bip-tap-vm.mediawiki

```diff
@@ -80,6 +80,10 @@ follows:
 # Initialize a new empty MS-SMT tree as specified in [[./bip-tap-ms-smt.mediawiki|bip-tap-ms-smt]].
 # For each Taproot Asset input ''c_i'', identified in the <code>prev_asset_witnesses</code> field:
 ## Serialize the referenced previous asset leaf (identified by <code>prev_outpoint || prev_asset_id || prev_asset_script_hash</code>) in TLV format.
+### For a minting transaction, a copy of the output leaf with emptied <code>prev_asset_witnesses</code> is used, in addition to these modifications:
+#### If the minted asset has a group key, the <code>asset_script_hash</code> of the copied leaf should be set equal to the group key. This enforces that the state transition verification uses the group key when validating the spend.
```

Comment:
Same here re generally cleaning up `script_hash` vs `script_key` across the spec

Reply:
> of the copied leaf should be set equal to the group key.

Hmm, I think we still want to commit to this `script_key` value so validation context is bound completely. Otherwise a sig can re re-used for each leaf item? 

In other words, we can have validate be based on `asset_group_key`, while serializing `asset_script_key` in the TLV leaf. This way both are still committed to, and we have a single branch that uses `asset_group_key` when this case is detected.

Reply:
The modification only affects the referenced _previous leaf_, that is used to construct the previous outpoint for the virtual tx. The output leaf (still committing to `asset_script_key`) remains the same.

Or did I misunderstand your comment?